### ISSUE TITLE
Community call - 2024.04.25

### ISSUE THREAD

**Issue by gdalle** [JuliaGraphs/JuliaGraphs-meta#16]
- See #15 cause no one was there
- How to fund JuliaGraphs?

**Comment by Krastanov** [comment]
For funding, it makes a lot of sense to consider the NumFocus affiliated projects. It might not provide funding directly, but it will make it easy to have someone take care of bank accounts and centralizing the handling of money.

I do not have the bandwidth to be significant contributor here, but happy to help with the NumFocus application (this year I joined their committee, so I will have to recuse myself from voting, but that should be fine) and happy to help with reviews of the GraphOptim surgery (although take my review as a grain of salt, as I am not familiar with this codebase). I generally will not be available to join the monthly meetings in the foreseeable future.

**Comment by gdalle** [comment]
Yeah sorry about GraphsOptim, I've been completely underwater trying to give birth to DifferentiationInterface, but the baby is almost out

**Comment by Krastanov** [comment]
No worries, please take your time and do not burn out -- you are way too valuable of a resource :D

**Comment by just-cameron** [comment]
I was at the meeting, here's a few of my notes.

We chatted about a possible graph database built in Julia, and what would it take to make a graph database work?

## Needs

- Graphs with arbitrary metadata (i.e. https://github.com/JuliaGraphs/MetaGraphsNext.jl)
- Nodes indexable by UUID or another arbitrary measure, such as a `String`.
- Multigraphs with metadata, i.e. want to be able to support `A - knows - B` and `A - is sister of B`. 
- Owned graphs -- i.e. you and I have our own graphs with permissions. You can link to my public nodes but not to my private ones. Not sure what that'd look like

## Things I'll do

- I'm going to be playing around with this to see if we can get a super simple prototype up and running, @jpsamaroo suggested he might liked to be involved to add distributed compute support.
- Build a very simple problem to find pain points.

Thanks for having me y'all! Looking forward to maybe poking around the graphs world more.
